src/main/java/org/example/homework8/Bedroom/Ceiling.java

```diff
@@ -0,0 +1,17 @@
+package org.example.homework8.Bedroom;
+
+public class Ceiling {
+    int height;
+    int paintedColour;
+    public Ceiling(int height, int paintedColour) {
+        this.height = height;
+        this.paintedColour = paintedColour;
+    }
+
+    public int getHeight(){
+        return this.height;
```

Comment:
the same about this in getters, here and below

Reply:
Improved.